architecture/infrastructure_components/kubernetes_infrastructure.adoc

```diff
@@ -74,6 +73,51 @@ use the `native` method but pre-configure your own load balancer of choice.
 endif::[]
 |===
 
+[[control-plane-static-pods]]
+=== Control Plane Static Pods
+
+Starting in {product-title} 3.10, the deployment model for installing and
+operating the core control plane components changed. Prior to 3.10, the API server
+and the controller manager components ran as stand-alone host processes operated
+by SystemD. In 3.10, these two components are moved to
```

Comment:
Seems should type 'systemd' instead of 'SystemD' here and in other places? See https://www.freedesktop.org/wiki/Software/systemd/ 'Spelling' part.

Reply:
Had not seen that. That's great, thanks!